release-notes-3.3.0.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
-#### New Features
+#### New Features  
+- [SCMOD-8484](https://portal.digitalsafe.net/browse/SCMOD-8484): Propagate failures through subtasks  
+Functionality has been added that allows for the failure of a task to propagate up through any tasks that are no longer able to run because the task has failed.  
```

Comment:
Is it worth mentioning that these are prerequisite to the task?

Reply:
these aren't the prerequisite to the task, the task that failed is the prerequisite to the ones involved in the roll up, but since it goes back up multiple levels it being described as affecting the tasks that will no longer ever be eligible to run due to the failure I think is better.